components/AddCarForm/AddCarList.js

```diff
@@ -0,0 +1,68 @@
+import Image from "next/image";
+import { StyledDescriptionList } from "../CarsDetails/StyledCarDetails";
+import styled from "styled-components";
+import {
+  StyledCompareButton,
+  StyledLikeButton,
+} from "../Favorites/StyledFavorites";
+
+const StyledAddCarList = styled.ul`
+  list-style: none;
+  padding: 0px;
+  gap: 10px;
+  display: grid;
+  font-size: 1em;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  border: 3px, solid, red;
```

Comment:
I don't think this works with `,`

```suggestion
  border: 3px solid red;
```